.github/workflows/auto-update-spec-repo-links.yml

```
@@ -77,5 +77,4 @@ jobs:
           git push --set-upstream origin $branch
           gh pr create --title "$message" \
```

```suggestion
          gh pr create --title "[chore] $message" \
```
This would also make it skip changelog verification.

---

https://github.com/open-telemetry/semantic-conventions/actions/runs/13010412553/job/36286799062
🤔

---

Ah, not a problem for the bot-opened PRs but for this one here:

Renaming does not trigger an event currently, so the check runs against the old name until one would synchronize (which trask will have to do anyway before merging):

https://github.com/open-telemetry/semantic-conventions/blob/5b24784e032ada99312d5a17d6970da434f5ce88/.github/workflows/changelog.yml#L9

We could also add the `edited` event to the workflow triggers:

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request

---

https://github.com/open-telemetry/semantic-conventions/pull/1837

---

nice!